COVID_HUNGARY_REPORT.wls

```diff
@@ -1541,7 +1027,7 @@ ColorData["Crayola"]["VividViolet"]},
 {Thickness[0.01],ColorData["Crayola"]["JungleGreen"]}};
 
 
-(* ::Subsection::Closed:: *)
+(* ::Subsection:: *)
 (*Seasonality*)
```

Comment:
As I see in the paper, seasonality plot is one of the 4 subfigures in Fig. 10. I would recommend to 
- change Seasonality top level header to "Seasonality - Plots for Figure 10",
- remove "Plots for Figure 10" cell
- make "Seasonality" cell to the same level as "... control" (which should be renamed, see comment below)